src/xrUICore/Windows/UIWindow.h

```diff
@@ -142,34 +142,34 @@ class XRUICORE_API CUIWindow : public CUISimpleWindow
     WINDOW_LIST m_ChildWndList;
 
     //указатель на родительское окно
-    CUIWindow* m_pParentWnd;
+    CUIWindow* m_pParentWnd{nullptr};
 
     //дочернее окно которое, захватило ввод мыши
-    CUIWindow* m_pMouseCapturer;
+    CUIWindow* m_pMouseCapturer{nullptr};
 
     //дочернее окно которое, захватило ввод клавиатуры
-    CUIWindow* m_pKeyboardCapturer;
+    CUIWindow* m_pKeyboardCapturer{nullptr};
 
     //кому шлем сообщения
-    CUIWindow* m_pMessageTarget;
+    CUIWindow* m_pMessageTarget{nullptr};
 
     // Последняя позиция мышки
     Fvector2 cursor_pos;
 
     //время прошлого клика мышки
     //для определения DoubleClick
-    u32 m_dwLastClickTime;
-    u32 m_dwFocusReceiveTime;
+    u32 m_dwLastClickTime{0};
+    u32 m_dwFocusReceiveTime{0};
 
     //флаг автоматического удаления во время вызова деструктора
-    bool m_bAutoDelete;
+    bool m_bAutoDelete{false};
 
     // Is user input allowed
-    bool m_bIsEnabled;
+    bool m_bIsEnabled{false};
 
     // Если курсор над окном
-    bool m_bCursorOverWindow;
-    bool m_bCustomDraw;
+    bool m_bCursorOverWindow{false};
+    bool m_bCustomDraw{false};
```

Comment:
Everything should be initialized using `{}`.
```suggestion
    bool m_bCustomDraw{};
```